### ISSUE TITLE
1.2.9 update broke the whole plugin

### ISSUE THREAD

**Issue by Pilvinen** [AdvancedPlugins/Seasons#102]
### Describe the bug

Spigot 1.20.6
Latest dev version of ProtocolLib.

No errors, but I'm just getting:
```
[Server thread/INFO]: [AdvancedSeasons] Disabling AdvancedSeasons v1.2.9
```
When the server has finished starting up. No clues anywhere why this might be happening.

Honestly, ramp up your game. I paid 20$ for this plugin and I haven't yet had a single chance to use it because it's so broken on Spigot. All my times goes into filing bug reports. You should test your software before releasing new versions.

### How to reproduce

Install Spigot 1.20.6 and AdvancedSeasons 1.2.9. I guess. That's the only thing I did.

**Comment by udei6** [comment]
same thing here

**Comment by homerek111** [comment]
same thing

**Comment by RyanZuZi** [comment]
> ### Describe the bug
> Spigot 1.20.6 Latest dev version of ProtocolLib.
> 
> No errors, but I'm just getting:
> 
> ```
> [Server thread/INFO]: [AdvancedSeasons] Disabling AdvancedSeasons v1.2.9
> ```
> 
> When the server has finished starting up. No clues anywhere why this might be happening.
> 
> Honestly, ramp up your game. I paid 20$ for this plugin and I haven't yet had a single chance to use it because it's so broken on Spigot. All my times goes into filing bug reports. You should test your software before releasing new versions.
> 
> ### How to reproduce
> Install Spigot 1.20.6 and AdvancedSeasons 1.2.9. I guess. That's the only thing I did.
> 
> ### Screenshots / Videos
> _No response_
> 
> ### Server Log
> _No response_

Hey Pilvinen, I feel you, man. It's such a bummer that I spent money and can't even use it for 1.20.6. Hopefully, they fix it real soon.

**Comment by babyonyt** [comment]
I'm a bit behind, but I'm encountering the same problem with version 1.20.6. I tend to be cautious about purchasing plugins because, inevitably, they never seem to work right off the bat (as expected by now).

**Comment by RyanZuZi** [comment]
> I'm a bit behind, but I'm encountering the same problem with version 1.20.6. I tend to be cautious about purchasing plugins because, inevitably, they never seem to work right off the bat (as expected by now).

How can a developer release an update without testing it first lol?

**Comment by babyonyt** [comment]
> > I'm a bit behind, but I'm encountering the same problem with version 1.20.6. I tend to be cautious about purchasing plugins because, inevitably, they never seem to work right off the bat (as expected by now).
> 
> How can a developer release an update without testing it first lol?

I'm certain they did, but apparently not thoroughly enough.

**Comment by Pilvinen** [comment]
Seems to be mostly fixed in 1.3.0.  I still get sometimes some kind of ... authentication server ? issue where AdvancedSeasons unloads itself. But that has happened only once so far.

**Comment by GC-spigot** [comment]
> > I'm a bit behind, but I'm encountering the same problem with version 1.20.6. I tend to be cautious about purchasing plugins because, inevitably, they never seem to work right off the bat (as expected by now).
> 
> 
> 
> How can a developer release an update without testing it first lol?

We test each and every update, the problem was caused after with the publishing system in place rather than the actual plugin. We've put changes in place to make sure problems like these do not happen again.

**Comment by GC-spigot** [comment]
> Seems to be mostly fixed in 1.3.0.  I still get sometimes some kind of ... authentication server ? issue where AdvancedSeasons unloads itself. But that has happened only once so far.

That may happen if your server loses internet connection for a longer period of time. 



This issue is now solved with latest update. My apologies everyone!
